source/winAPI/constants.py

```diff
@@ -3,9 +3,14 @@
 # This file is covered by the GNU General Public License.
 # See the file COPYING for more details.
 
+import winVersion
 import enum
 
 
+#: Verification of the architecture of the running system
+is_64Bit = winVersion.getWinVer().processorArchitecture.endswith("64")

```

Comment:
We use capitals when naming constants

```suggestion
IS_64_BIT: bool = winVersion.getWinVer().processorArchitecture.endswith("64")
```